core/api/src/app/users/get-user-language.ts

```diff
@@ -0,0 +1,11 @@
+import { NotificationsService } from "@/services/notifications"
+
+export const getUserLanguage = async (
```

Comment:
I think this is not a proper approach. Language is not only used in notifications and from the domain pov is part of the user not notifications.

if notification sub system needs the language we will need to pass it through notification request or persist it too when the user change the language